### PR TITLE
EZP-30882: Exposed Thumbnail in REST Version

### PR DESCRIPTION
**Pull request by Nattfarinn**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-30882](https://jira.ez.no/browse/EZP-30882)
| **Type**| feature
| **Target version** | eZ Platform `v3.0.0`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | yes

## Depends on: https://github.com/ezsystems/ezpublish-kernel/pull/2868

**TODO**:
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: src/lib/Server/Output/ValueObjectVisitor/Version.php [hunk 1]
```diff
@@ -121,4 +128,28 @@ protected function visitVersionAttributes(Visitor $visitor, Generator $generator
             )
         );
     }
+
+    private function visitThumbnail(
+        Visitor $visitor,
+        Generator $generator,
+        ?Thumbnail $thumbnail
+    ) {
```
Comment (adamwojs) [inline src/lib/Server/Output/ValueObjectVisitor/Version.php]:
```suggestion
    ): void {
```

**Comment by micszo** [comment]
All tests [passed](https://travis-ci.org/ezsystems/ezplatform-rest/builds/627633008?utm_source=github_status&utm_medium=notification).

File: src/lib/Server/Output/ValueObjectVisitor/Version.php [hunk 2]
```diff
@@ -42,11 +43,17 @@ public function __construct(FieldTypeSerializer $fieldTypeSerializer)
      */
     public function visit(Visitor $visitor, Generator $generator, $data)
     {
+        /** @var \eZ\Publish\Core\Repository\Values\Content\Content $content */
```
Comment (alongosz) [inline src/lib/Server/Output/ValueObjectVisitor/Version.php]:
```suggestion
        /** @var \eZ\Publish\API\Repository\Values\Content\Content $content */
```

File: tests/bundle/Functional/xsd/Version.xsd [hunk 3]
```diff
@@ -106,6 +106,15 @@
             </xsd:simpleContent>
           </xsd:complexType>
         </xsd:element>
+        <xsd:element name="Thumbnail">
```
Comment (alongosz) [inline tests/bundle/Functional/xsd/Version.xsd]: cc @ezsystems/documentation-team - REST spec for Version object to be updated - we have a new property.

**Pull request merged**